src/Label/label.stories.js

```diff
@@ -2,4 +2,68 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import Label from "./index";
 
-storiesOf("Label", module).add("with content", () => <Label label="Test" />);
+storiesOf("Label", module).add("simple label", () => 
+    <Label label="Test" />
+);
+
+storiesOf("Label", module).add("with short tooltip", () =>
+    <div style={{paddingTop:100,paddingLeft:100}}>
+        <Label 
+            label="Some field label:" 
+            tooltipMessage="This is a required field"
+            tooltipPlace="top"        
+        />
+    </div>
+);
+
+storiesOf("Label", module).add("with short tooltip on right", () =>
+    <div style={{paddingTop:100,paddingLeft:100}}>
+        <Label 
+            label="Some field label:"   
+            labelType="inline"          
+            tooltipMessage="This is a required field"
+            tooltipPlace="top"     
+            style={{floar:"left"}}   
```

Comment:
typo

Reply:
Correct, also, I gonna take it out, not needed since the label is inline here...